src/components/Header/Header.css

```diff
@@ -5,25 +5,31 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  color: white;
+  font-size: 10px;
 }
 
 .list-navigation {
   display: flex;
   column-gap: 20px;
   list-style-type: none;
-  color: white;
-  font-size: 10px;
 }
 
 .loupe * {
   fill: white;
 }
 
 .loupe {
-  padding-right: 40px;
   cursor: pointer;
 }
 
 .list-navigation li {
   cursor: pointer;
 }
+
+.right-icon {
+  display: flex;
+  column-gap: 20px;
+  padding-right: 40px;
```

Comment:
why do you need this property?

Reply:
becouse this is flex box, so that the elements are located on the horizontal axis,  space between them and an indent on the right.